sdk/synapse/synapse-artifacts/src/artifactsClientContext.ts

```diff
@@ -40,44 +37,29 @@ export class ArtifactsClientContext extends coreHttp.ServiceClient {
     if (!options) {
       options = {};
     }
+    const defaults: ArtifactsClientOptionalParams = {
+      requestContentType: "application/json; charset=utf-8",
+      credential: credentials
+    };
 
-    if (!options.userAgent) {
-      const defaultUserAgent = coreHttp.getDefaultUserAgentValue();
-      options.userAgent = `${packageName}/${packageVersion} ${defaultUserAgent}`;
-    }
+    const packageDetails = `azsdk-js-synapse-artifacts/1.0.0-beta.5`;
```

Comment:
please add this to the `constantsPath` list in `package.json`.

Reply:
@sarangan12 please consider generating the `constantsPath` list automatically by code gen.